hadoop-ozone/dist/src/main/compose/upgrade/upgrades/non-rolling-upgrade/1.1.0-1.2.0/callback.sh

```diff
@@ -64,8 +64,7 @@ with_old_version_downgraded() {
 
 with_new_version_finalized() {
   _check_hdds_mlvs 2
-  # OM currently only has one layout version.
-  _check_om_mlvs 0
```

Comment:
@smengcl @errose28 maybe I'm missing something, but wouldn't this fail if we actually ran upgrade path test?

https://github.com/adoroszlai/hadoop-ozone/runs/6613025379#step:5:738

Reply:
You are right.

Though in the latest feature branch it is indeed still 0 (probably fixed when I was resolving conflicts when doing the merge master 3 days ago):

https://github.com/smengcl/hadoop-ozone/blob/HDDS-4944/hadoop-ozone/dist/src/main/compose/upgrade/upgrades/non-rolling-upgrade/1.1.0-1.2.0/callback.sh#L47-L49

Not sure why I bumped it when doing this PR.

Reply:
Hmm. When checking the actual PR merged into the multi-tenancy feature branch I don't even see this line of diff in `pre_finalized`:

https://github.com/apache/ozone/commit/83a8c5f14b28b03d7f852bbde1dd34610b4c67a0

Reply:
Anyway, the om version bump in `with_new_version_finalized` is intended. As after the finalization it should be bumped to the latest version. But in `pre_finalized` it should still be zero.

Reply:
But this callback applies when new version is 1.2.0.

Reply:
I was going to say when I was merging master to the feature branch just 3 days ago, I had to bump the `_check_om_mlvs` further to 3 to pass acceptance (misc):

https://github.com/apache/ozone/blame/HDDS-4944/hadoop-ozone/dist/src/main/compose/upgrade/upgrades/non-rolling-upgrade/1.2.1-1.3.0/callback.sh#L74

But then I realized this is the one from 1.1.0 to 1.2.0.

Yup so in this case it should still be zero (in 1.2.0).

Reply:
Addressed this in commit: https://github.com/apache/ozone/pull/3450/commits/00a500786f0d6112af24b8b1c8100e9e0add1f28